### PR TITLE
Plat UI 3141 create seperate browser journey test repo

### PR DESCRIPTION
**Pull request by goodvibes**
https://jira.tools.tax.service.gov.uk/browse/PLATUI-3141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,22 +40,19 @@ http://localhost:9240/help/cookie-details
...
-## Running the UI acceptance tests
+Accessibility tests are run in Jenkins or locally via the journey tests.
 
-To execute the UI acceptance tests on your local machine, ensure that you have [local-selenium-grid](https://github.com/hmrc/local-selenium-grid) installed and running. Once set up, you can proceed with running the tests with.
-```
-./run_acceptance_tests.sh
-```
+## UI Journey Tests
+
+UI Journey Testscan be found at https://github.com/hmrc/help-frontend-ui-tests
```
Comment (TimothyFothergill) [inline README.md:55]: Small typo here `Testscan`

Comment (goodvibes) [reply README.md]: Fixed 👍

**Review by TimothyFothergill — DISMISSED** [review]
lgtm

**Review by TimothyFothergill — APPROVED** [review]
lgtm

**Pull request merged**